### ISSUE TITLE
Content editing toolbar is obscured when text is increased to 200% size

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov#574]
## Steps to replicate
* As content editor you should have a menu at the top with View, Edit, Revision, Children, Preview Link, Scheduled Transitions
* page is set to 100% zoom
* Text size is increased to 200% using the Zoom Text Only Chrome extension

## Expected result
You can make text at least 200% larger without losing content / functionality.

## Actual result
Top menu is obscured and can not be seen

![image](https://github.com/localgovdrupal/localgov/assets/19519457/b97d0649-1501-46da-bcf1-29b07a1ccff0)

## WCAG reference
WCAG 2.1 SC 1.4.4 https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html

**Comment by markconroy** [comment]
I'll have to test this some more. At 400% I still see the tabs.
![Screenshot 2023-07-07 at 15 32 26](https://github.com/localgovdrupal/localgov/assets/2183332/33ddaa95-6eed-4178-bde6-d021e415c08b)

**Comment by keelanfh** [comment]
To note, this is specifically about increasing the text size, not zoom.
